changelogs/unreleased/1249-fix-race-condition-compile-requests.yml

```diff
@@ -0,0 +1,8 @@
+---
+description: Add support to the compilerservice to request a compile that is part of a database transaction.
+issue-nr: 1249
+issue-repo: inmanta-lsm
+change-type: patch
```

Comment:
I would label this a minor, and bump the version now so we can depend on the right increment in inmanta-lsm.